core/sql/cli/CliExtern.cpp

```diff
@@ -6316,7 +6316,8 @@ Lng32 SQL_EXEC_DeleteHbaseJNI()
       threadContext->incrNumOfCliCalls();
 
       HBaseClient_JNI::deleteInstance();
-      HiveClient_JNI::deleteInstance();
+      // The Hive client persists across connections
+      // HiveClient_JNI::deleteInstance();
```

Comment:
Whenever a context is dropped, both hbaseClient_JNI and hiveClient_JNI is getting deleted. This CLI call is being called from the two places shown below. I am not sure about the need for this calls in these functions. If it is needed, I would think there would be similar problem with HBaseClient_JNI::deleteInstance too. Or is it better to get rid of this CLI call.

arkcmp/CmpStatement.cpp:  SQL_EXEC_DeleteHbaseJNI();
sqlcomp/CmpSeabaseDDLcommon.cpp:  SQL_EXEC_DeleteHbaseJNI();

In case of mxosrvr, there is default context. In case of T2 driver, the CliContext would be deallocated and these objects would be deleted. I am assuming that it should be ok to do that.